source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -369,7 +375,7 @@
                         }
                         valuesByItem.hasComments = ko.computed(function () {
                             return valuesByItem.comments.find(function (comment) {
-                                return !!comment.HasComment();
+                                return !!comment.HasComment() && !!comment.isSelfComment;
```

Comment:
you can also return double negation to [_computed return_](https://github.com/CommonJobs/CommonJobs/pull/481/files#diff-1b28b31e2207786095b5affe916c595bR377) to ensure to return a boolean in place of a comment. Or, better yet, maybe _contains_ is the right underscore function, see http://underscorejs.org/#contains